Streaming/src/main/java/io/straas/android/sdk/streaming/demo/OverlayLayout.java

```diff
@@ -0,0 +1,99 @@
+package io.straas.android.sdk.streaming.demo;
+
+import android.content.Context;
+import android.graphics.PixelFormat;
+import android.view.Gravity;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.WindowManager;
+import android.widget.FrameLayout;
+
+public abstract class OverlayLayout extends FrameLayout implements View.OnClickListener {
+
+    interface Listener {
+        void onMove(OverlayLayout overlayLayout);
+        void onStartClick();
+        void onDestroyClick();
+    }
+
+    private WindowManager.LayoutParams mLayoutParams;
+    protected final Listener mListener;
+
+    protected WindowManager.LayoutParams createLayoutParams(Context context) {
+        final WindowManager.LayoutParams params = new WindowManager.LayoutParams(getLayoutWidth(context), ViewGroup.LayoutParams.WRAP_CONTENT,
+                WindowManager.LayoutParams.TYPE_SYSTEM_ERROR, WindowManager.LayoutParams.FLAG_NOT_FOCUSABLE |
```

Comment:
params copied from [telecine](https://github.com/JakeWharton/Telecine/blob/master/telecine/src/main/java/com/jakewharton/telecine/OverlayView.java#L49), and explained [here](http://liball.me/windowmanagerde-na-xie-dan-teng-de-shi/)